api/krusty/fnplugin_test/fnexectest.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# not sure if we want to generate bash scripts, since we always want to run
+# only trusted executables
```

Comment:
Saying 'trusted executable' here just raises questions as to what is trusted.
Suggest dropping this comment.

Please just comment saying that this script is used as a generator function in the test.